wordpress-mobile/WordPress-iOS
# Signup Epilogue: adjust table width for iPad

Ref #13939 

This updates the Signup Epilogue's table width based on the size class and device orientation.
- For the Regular x Regular size class:
  - In portrait, the table is 66% of the view width.
  - In landscape, the table is 50% of the view width.
- For all other size classes, the table is 100% of the view width.

In addition, the Done button left/right margins always line up with the table's content.

To test:

---
- Signup on an iPad. (See below to avoid actually signing up)
- Verify the table and button width are sized properly in portrait and landscape.

| ![ipad_port](https://user-images.githubusercontent.com/1816888/80648577-a3884680-8a2d-11ea-9362-c59fba81882e.png) | ![ipad_land](https://user-images.githubusercontent.com/1816888/80648590-a84cfa80-8a2d-11ea-8e1f-050267fa7747.png) |
|--------|-------|

---
- Signup on an iPhone, or multitask on an iPad and change the view size.
- Verify the table and button width are sized properly.

![iphone](https://user-images.githubusercontent.com/1816888/80637245-e7725000-8a1b-11ea-894c-86febeea76c6.png)

---
To show the Signup Epilogue without actually signing up:

- Check out the accompanying `WordPressAuthenticator` branch locally.
- In WPiOS `Podfile`, point `WordPressAuthenticator` to your local Auth.
- In `LoginViewController`, change `showLoginEpilogue` to show the signup epilogue:

```
func showLoginEpilogue(for credentials: AuthenticatorCredentials) {
    guard let navigationController = navigationController else {
        fatalError()
    }
    
    showSignupEpilogue(for: credentials)
    
//        authenticationDelegate.presentLoginEpilogue(in: navigationController, for: credentials) { [weak self] in
//            self?.dismissBlock?(false)
//        }
}
```

- _Login_ with an existing account, and the _signup_ epilogue will be displayed.

---
PR submission checklist:
